src/ng-at-internet.provider.js

```diff
@@ -242,13 +267,14 @@ angular.module("ng-at-internet")
                         updateData(pageData);
 
                         if (pageData.name) {
-
                             var customVars = {};
                             _.forOwn(AT_INTERNET_CUSTOM_VARS, function (conf, attr) {
                                 updateCustomVars(config.defaults, conf, attr, customVars);
                             });
                             pageData.customVars = customVars;
-
+                            if (config.identifiedVisitor !== {}) {
```

Comment:
this test is always true. {} === {} is false. This is reference comparison.
You can use : 
- lodash ```_.isEmpty(config.identifiedVisitor)```
- or angular ```angular.isEqual()```
for making this kind of test